src/runtime/lpython_builtin.py

```diff
@@ -732,3 +732,12 @@ def _lpython_str_startswith(s: str ,sub: str) -> bool:
     if res:
         res = res and (j == len(sub))
     return res
+
+
+def list(s: str) -> list[str]:
+    l: list[str]
+    l = []
+    i: i32
+    for i in range(len(s)):
+        l.append(s[i])
+    return l
```

Comment:
Ah I see, it's doing the runtime as well. So we should test both compile time and runtime.

For runtime, should we rename this function to `_lcompilers_str_to_list`, just to make it clear that this is an internal function? 

CC @czgdp1807.

Reply:
I think `list` is more approriate because it is a builtin function or intrinsic in python much like `ord`, `hex`, `bin` etc. and we can handle it the same way.